src/en/howto-applicationupdateseries.md

```diff
@@ -0,0 +1,113 @@
+Title: Change an application' series post deployment
```

Comment:
since this is really about how to do it pre-2.3 maybe title this in that way? Otherwise just use the normal documentation? So this turns into "how to change an application series pre-juju2.3" or something ?